include/alpaka/mem/view/Traits.hpp

```diff
@@ -223,9 +223,6 @@ namespace alpaka
         static_assert(
             meta::IsIntegralSuperset<SrcIdx, ExtentIdx>::value,
             "The source view and the extent are required to have compatible index types!");
-        static_assert(
```

Comment:
Should we not keep this assert?

Reply:
No because the same assert is also on line 217 :)